src/main.py

```diff
@@ -83,31 +86,31 @@ def main(module_name: OracleModule):
     logger.info({'msg': 'Sanity checks.'})
     check_providers_chain_ids(web3)
 
-    if module_name == OracleModule.ACCOUNTING:
+    if module == OracleModule.ACCOUNTING:
         logger.info({'msg': 'Initialize Accounting module.'})
         accounting = Accounting(web3)
         accounting.check_contract_configs()
         accounting.run_as_daemon()
-    elif module_name == OracleModule.EJECTOR:
+    elif module == OracleModule.EJECTOR:
         logger.info({'msg': 'Initialize Ejector module.'})
         ejector = Ejector(web3)
         ejector.check_contract_configs()
         ejector.run_as_daemon()
-    elif module_name == OracleModule.READINESS:
+    elif module == OracleModule.READINESS:
         logger.info({'msg': 'Check oracle is ready to work in the current environment.'})
         readiness = ReadinessModule(web3)
         sys.exit(readiness.execute_module())
 
 
-def check_required_variables():
+def check_required_variables(module_name: OracleModule):
     errors = []
     if '' in variables.EXECUTION_CLIENT_URI:
         errors.append('EXECUTION_CLIENT_URI')
     if variables.CONSENSUS_CLIENT_URI == '':
         errors.append('CONSENSUS_CLIENT_URI')
     if variables.KEYS_API_URI == '':
         errors.append('KEYS_API_URI')
-    if variables.LIDO_LOCATOR_ADDRESS in (None, ''):
+    if variables.LIDO_LOCATOR_ADDRESS in (None, '') and module_name != OracleModule.READINESS:
```

Comment:
Maybe create separate flow for node testing?

Reply:
fixed, take a look at